lib/config/wdtk-routes.rb

```diff
@@ -5,6 +5,8 @@
   get '/scotland' => redirect('/body?tag=scotland', status: 302)
   get '/cymru' => redirect('/body?tag=wales', status: 302)
   get '/wales' => redirect('/body?tag=wales', status: 302)
+  get '/ni' => redirect('/body?tag=ni', status: 302)
+  get '/northernireland' => redirect('/body?tag=ni', status: 302)
```

Comment:
I'd make this `/northern-ireland`. Happy to tweak when I merge it if you are?

Reply:
> I'd make this `/northern-ireland`. I'm happy to tweak when I merge it if you are?

Good point - I've pushed a patch 😊